src/Renderer.js

```diff
@@ -170,7 +170,7 @@ export class Renderer {
     if (href === null) {
       return text;
     }
-    let out = '<a href="' + escape(href) + '"';
+    let out = '<a href="' + href + '"';
```

Comment:
What about when `href` contains a `'` or `"`? 

Do we have a test for that?

Reply:
the href runs through `encodeURI` in the `cleanURL`

https://github.com/UziTech/marked/blob/57a203182cddb4dc1ae4ac8f7c72650eddf3442d/src/helpers.js#L98